File: src/com/kostu96/bots/entities/animals/Wolf.java
package com.kostu96.bots.entities.animals;

import com.kostu96.bots.World;
import com.kostu96.bots.entities.Entity;
import com.kostu96.bots.utils.ImageManager.ImageID;
import com.kostu96.bots.utils.Point;

public class Wolf extends Animal {
	private static final int STRENGTH = 9;
	private static final int INITIATIVE = 5;

	public Wolf(World world, Point position) {
		super(world, position, STRENGTH, INITIATIVE);
	}

	@Override
	public void collision(Entity other) {

	}

	@Override
	public ImageID getImageID() {
		return ImageID.WOLF;
	}

}

File: src/entities/wolf.py
from .animal import Animal

class Wolf(Animal):
    color = (100, 110, 110)
    symbol = 'WF'

    def __init__(self, world, position):
        super().__init__(world, position, 9, 5)

    def __str__(self):
        return 'Wolf'

    def getColor(self):
        return Wolf.color

    def getSymbol(self):
        return Wolf.symbol
